website/docs/docs-section/appendix/contract-costs.md

```diff
@@ -103,6 +103,7 @@ export const fetchGasCostData = async () => {
 
 export const StyledTag = ({ children }) => (
   <span
+    class="styled-tag"
```

Comment:
I think you've got a typo here. You probably wanted to use `className` instead of `class`.